table/block_based/block_based_table_reader.cc

```diff
@@ -2137,6 +2206,10 @@ Status BlockBasedTable::MaybeReadBlockAndLoadToCache(
   char compressed_cache_key[kMaxCacheKeyPrefixSize + kMaxVarint64Length];
   Slice key /* key to the block cache */;
   Slice ckey /* key to the compressed block cache */;
+  bool is_cache_hit = false;
+  bool no_insert = true;
+  size_t usage = 0;
```

Comment:
Let's limit the scope of usage/nkeys (seems to me they're only referenced in the if (block_cache_tracer_ && ...) block below).

Reply:
Sure. I moved them to where they are used.